commands/keplr.js

```diff
@@ -224,6 +239,17 @@ const keplr = {
     return true;
   },
 
+  async getWalletAddress() {
+    await playwright.switchToKeplrWindow();
+    await module.exports.goToHome();
+    const page = await playwright.keplrWindow();
+    await playwright.waitAndClickByText(notificationPageElements.copyAddress);
+    await page.click(notificationPageElements.copyWalletAddressSelector);
+    const walletAddress = clipboardy.readSync();
```

Comment:
why would we want to store the walletAddress globally? don't we have multiple addresses, depending on our chain?

also remove const to avoid variable shadowing

Reply:
At one time, we will be dealing with one instance of the wallet. And for that instance there will be only only one wallet address for the agoric chain